rest-api/dao/participant_summary_dao.py

```diff
@@ -403,12 +403,53 @@ def update_from_biobank_stored_samples(self, participant_id=None):
 
     sample_sql = replace_null_safe_equals(sample_sql)
     counts_sql = replace_null_safe_equals(counts_sql)
+
+
     with self.session() as session:
       session.execute(sample_sql, sample_params)
       session.execute(counts_sql, counts_params)
       session.execute(enrollment_status_sql, enrollment_status_params)
+      # TODO: Change this to the optimized sql in _update_dv_stored_samples()
```

Comment:
is this a todo for another pull request later or a lingering comment?

Reply:
There is no ticket yet, but its on a lot of minds that the biobank nightly processing job is painfully inefficient and occasionally fails because of GCP timing something out. A large part of that stems from non-optimized SQL statements.